extensions/spark/kyuubi-spark-authz/src/test/scala/org/apache/kyuubi/plugin/spark/authz/gen/TableCommands.scala

```diff
@@ -410,6 +410,26 @@ object TableCommands {
     TableCommandSpec(cmd, Seq(tableDesc), DESCTABLE)
   }
 
+  val DescribeRelationTable = {
+    val cmd = "org.apache.spark.sql.catalyst.plans.logical.DescribeRelation"
+    val columnDesc = ColumnDesc("partitionSpec", classOf[PartitionColumnExtractor])
+    val tableDesc = TableDesc(
+      "relation",
+      classOf[ResolvedTableTableExtractor],
+      Some(columnDesc),
+      isInput = true,
+      setCurrentDatabaseIfMissing = true)
+    TableCommandSpec(cmd, Seq(tableDesc), DESCTABLE)
```

Comment:
You could put multiple table specs for different spark version when constructing the TableCommandSpec here rather than separate it into two different commands. They then will be used one by one in PriviligeBuilder to extract tables from the command.

Reply:
ah ok i better understand how it's working thanks :)

Reply:
Yeah, we are doing this to be decoupled from the differed details from different Spark versions. Good to put the spec for the newer spark version at the front. Plus, you can now add the unit test for the lower spark versions now.

Reply:
Now that i better how intropection work, i found a way to have a one command work for all spark 3 version. 
Thanks @bowenliang123  for the help and the explanation.

Reply:
No problem. The new serde design in 1.7 is a much more elegant way than the way as hardcoded mapping in 1.6, which is worth more focus in the specs' declaration. Let us know if there's any progress or obstocles.

Reply:
hi @bowenliang123,
I found why 3.0 test was not working, because V2JdbcTableCatalogRangerSparkExtensionSuite can only work if it's 3.1 or higher so i add isSparkV31OrGreater

Reply:
yeah, but we still do not have any ut for "describe table" for 3.0, can you add a test inside `RangerSparkExtensionSuite` or its subclass?